doc/classes/RenderingServer.xml

```diff
@@ -1931,6 +1931,13 @@
 				Sets the visibility range values for the given geometry instance. Equivalent to [member GeometryInstance3D.visibility_range_begin] and related properties.
 			</description>
 		</method>
+		<method name="instance_reset_interpolation">
+			<return type="void" />
+			<param index="0" name="instance" type="RID" />
+			<description>
+				Resets motion vectors and other interpolated values.
```

Comment:
```suggestion
				Resets motion vectors and other interpolated values. Use this [i]after[/i] teleporting a mesh from a position to another to avoid ghosting artifacts.
```